GRIB2_Template_4_50_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,22 @@
+Title_en,OctetNo,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+analysis or forecast at a horizontal level or in a horizontal layer at a point in time for aerosol,10,Parameter category,(see Code table 4.1),,4.1,,Operational
+analysis or forecast at a horizontal level or in a horizontal layer at a point in time for aerosol,11,Parameter number,(see Code table 4.2),,4.2,,Operational
+analysis or forecast at a horizontal level or in a horizontal layer at a point in time for aerosol,12-13,Aerosol type,(see Code table 4.233),,4.233,,Operational
```

Comment:
@sebvi -  the proposal says that Octets 12-13 should be: 
Constituent Type (see common code table C-14)

Reply:
constituent type and aerosol type are (almost) the same thing as that they both refer to an entry in Common Code Table 14. But in the ancient times, before the Common code table era, they were 2 separate tables 4.230 and 4.233. Since the template is for aerosol, it should be "Aerosol Type", so I corrected the issue but the csv file is correct